test/components/truncate_test.rb

```diff
@@ -17,9 +17,22 @@ def test_renders_truncated_span
     assert_selector("span.css-truncate.css-truncate-overflow", text: "content")
   end
 
+  def test_renders_truncated_p
+    render_inline(Primer::Truncate.new(tag: :p)) { "content" }
+
+    assert_selector("p.css-truncate.css-truncate-overflow", text: "content")
+  end
+
+  def test_renders_truncated_strong
+    render_inline(Primer::Truncate.new(tag: :strong)) { "content" }
+
+    assert_selector("strong.css-truncate.css-truncate-overflow", text: "content")
+  end
```

Comment:
what do you think of having a more generic test, so we don't have to keep adding a test every time we change a tag?

```rb
def test_renders_multiple_tags
  Primer::Truncate::TAG_OPTIONS.each do |tag|
    render_inline(Primer::Truncate.new(tag: tag)) { "content" }

    assert_selector("#{tag}.css-truncate.css-truncate-overflow", text: "content")
  end
end
```

Reply:
Yes! I like it!

Still learning a lot with Ruby. I will change to that one :)